.github/workflows/actions-publish.yml

```
@@ -37,13 +37,13 @@ jobs:
     runs-on: ubuntu-20.04
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: Read .nvmrc
-        run: echo ::set-output name=NVMRC::$(cat .nvmrc)
+        run: echo {NVMRC}={cat .nvmrc} >> $GITHUB_OUTPUT
```

This command is wrong, notice that NVRMC is being defined as value instead of variable. Due to this problem, the node version used in the pipeline is different from the node version in the .NVMRC file.

Maybe the suggestion below will work
```suggestion
        run: echo NVMRC={cat .nvmrc} >> $GITHUB_OUTPUT
```